drake/systems/Simulation.h

```diff
@@ -16,12 +16,14 @@ namespace Drake {
 
   // simulation options
   struct SimulationOptions {
-    double realtime_factor;  // 1 means try to run at realtime speed, < 0 is run as fast as possible
+    double realtime_factor;  // 1 means try to run at realtime speed, 0 is run as fast as possible, < 0 means use default
```

Comment:
Are we going to specify the integration method inside `SimulationOptions` in the future?